thetis/field_defs.py

```diff
@@ -46,6 +46,12 @@
     'unit': '',
     'filename': 'Tracer2d',
 }
+field_metadata['sediment_2d'] = {
+    'name': 'Depth averaged sediment',
```

Comment:
It's not necessarily depth-averaged, is it?

Reply:
Agreed - I have changed this and removed the depth-averaged word